static/app/gettingStartedDocs/java/log4j2.tsx

```diff
@@ -41,6 +41,48 @@ export const steps = ({
       "Install Sentry's integration with Log4j 2.x using either Maven or Gradle:"
     ),
     configurations: [
+      {
+        description: <h5>{t('Gradle')}</h5>,
+        configurations: [
+          {
+            description: (
+              <p>
+                {tct('For Gradle, add to your [code:build.gradle] file:', {
```

Comment:
Relevant for all integrations:
(m) Maybe add a quick (1 sentence) explainer that for gradle you just need to add the plugin and sentry and integrations will be automatically installed by the plugin.